packages/@orchidui-vue/src/Elements/AdditionalContent/DynamicType/OcDynamicType.vue

```diff
@@ -18,7 +18,10 @@ defineEmits(["addCustomer"]);
       v-for="(box, i) in boxes"
       :key="i"
       :class="
-        isCustomer ? '!grid grid-cols-4 grid-rows-2 gap-y-4 w-full !py-4' : ''
+        isCustomer
+          ? '!grid grid-cols-4 grid-rows-2 gap-y-4 w-full !py-4 ' +
+            (box?.style || '')
+          : ''
```

Comment:
tailwind does not work with dynamic class, Am I wrong?
```suggestion
        [
          isCustomer
            ? '!grid grid-cols-4 grid-rows-2 gap-y-4 w-full !py-4 '
            : '',
          box.class,
        ]
```